### PR TITLE
Implement picky-signtool

### PR DESCRIPTION
**Pull request by RRRadicalEdward**
picky-signtool uses picky Authenticode implementation and lief-rs for embedding an Authenticode signature to the target binary.

### CODE REVIEW

**Comment by awakecoding** [comment]
@RRRadicalEdward you can update this PR to point to the latest lief-rs master

**Comment by RRRadicalEdward** [comment]
> 
> 
> @RRRadicalEdward you can update this PR to point to the latest lief-rs master
Done

File: picky-signtool/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+            binary
+                .set_authenticode(certfile, private_key, Some(binary_name))
+                .map_err(|err| anyhow!("Failed to check signature: {}", err))?;
+
+            binary
+                .build(PathBuf::from(output_path), false)
+                .map_err(|err| anyhow!("Failed to build the signed executable: {}", err))?;
+
+            println!("Signed binary_name successfully!");
```
Comment (CBenoit) [inline picky-signtool/src/main.rs:97]: binary_name ?

**Review by CBenoit — CHANGES_REQUESTED** [review]
Thank you!
Here is my initial review. I'll probably have more suggestions related to the CLI API later.

File: picky-signtool/src/main.rs (hunk 1 above)
Comment (RRRadicalEdward) [reply picky-signtool/src/main.rs:97]: Shoud be `println!("Signed {} successfully!", binary_name);`

File: picky-signtool/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "picky-signtool"
+version = "0.1.0"
+authors = [ "Alexandr Yusuk <aleksandr.yusuk@apriorit.com>" ]
+edition = "2018"
+description = "A signtool like Authenticode sing and verify tool based on picky and lief-rs"
```
Comment (CBenoit) [inline picky-signtool/Cargo.toml:6]:
```suggestion
description = "A signtool like Authenticode sign and verify tool based on picky and lief-rs"
```

File: picky-signtool/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+                .help("Path where to save the signed binary")
+                .takes_value(true)
+                .validator(validate_executable_postfix)
+                .display_order(1),
+        )
+        .arg(
+            Arg::with_name(ARG_SIGN)
+                .short(ARG_SIGN)
+                .long("sign")
+                .help("Sing the input file")
```
Comment (CBenoit) [inline picky-signtool/src/main.rs:140]:
```suggestion
                .help("Sign the input file")
```

**Review by CBenoit — CHANGES_REQUESTED** [review]
Sing :stuck_out_tongue:

File: picky-signtool/Cargo.toml (hunk 2 above)
Comment (RRRadicalEdward) [reply picky-signtool/Cargo.toml:6]: I hate this typo😀

**Review by CBenoit — APPROVED** [review]
Thank you! I also added your name in `picky-asn1`, `picky-asn1-der`, `picky-asn1-x509` and `picky` crates's metadata as author.

**Pull request merged**